src/cclib/parser/gaussianparser.py

```diff
@@ -213,14 +214,24 @@ def extract(self, inputfile, line):
 
             if not hasattr(self, 'optdone'):
                 self.optdone = []
-
             self.optdone.append(len(self.geovalues) - 1)
 
+            if not hasattr(self, 'optstatus'):
+                self.optstatus = []
+            assert len(self.optstatus) > 0
```

Comment:
Shouldn't this throw an assertion if line 220 is ever called (i.e. ```optstatus``` isn't created until now)? Same with line 232.